sections/section-text-with-gallery.liquid

```diff
@@ -0,0 +1,297 @@
+{{ 'section-text-with-gallery.css' | asset_url | stylesheet_tag }}
+{{ 'slider-gallery.css' | asset_url | stylesheet_tag }}
+
+{%- assign _section = section.settings -%}
+{%- assign _blocks = section.blocks -%}
+
+<style>
+  .section-text-with-gallery--{{section.id}} {
+    padding-top: {{_section.padding_top}}px;
+    padding-bottom: {{_section.padding_bottom}}px;
+    background-color: {{_section.bg_color}};
+  }
+
+  .section-text-with-gallery--{{section.id}} .section__title {
+    font-size: {{_section.title_font_size}}px;
+  }
+
+  .section-text-with-gallery--{{section.id}} .section__inner {
+    align-items: {{_section.vertical_align}};
+  }
+
+  .section-text-with-gallery--{{section.id}} .section__content {
+    text-align: {{_section.text_align}};
+  }
+
+  @media(max-width: 767px) {
+    .section-text-with-gallery--{{section.id}} {
+      padding-top: {{_section.padding_top_mobile}}px;
+      padding-bottom: {{_section.padding_bottom_mobile}}px;
+    }
+
+    .section-text-with-gallery--{{section.id}} .section__title {
+      font-size: {{_section.title_font_size_mobile}}px;
+    }
+  }
+</style>
+
+<section class="section-text-with-gallery section-text-with-gallery--{{section.id}} {% if _section.reverse %} section-text-with-gallery--reverse {% endif %} {% if _section.full_width %} section-text-with-gallery--fullwidth {% endif %}">
+  <div class="shell">
+    <div class="section__inner">
+      <div class="section__content">
+        <div class="section__content-inner">
+          {% if _section.title != blank %}
+            <h2 class="section__title">
+              {{- _section.title -}}
+            </h2>
+            <!-- /.section__title -->
+          {% endif %}
+
+          {% if _section.content != blank %}
+            <div class="section__entry">
+              {{- _section.content -}}
+            </div>
+            <!-- /.section__entry -->
+          {% endif %}
+
+          {% if _section.btn_text != blank and _section.btn_url != blank %}
+            <div class="section__actions">
+              <a href="{{- _section.btn_url -}}" class="button">
+                {{- _section.btn_text -}}
+              </a>
+            </div>
+            <!-- /.section__actions -->
+          {% endif %}
+        </div>
+        <!-- /.section__content-inner -->
+      </div>
+      <!-- /.section__content -->
+
+      <div class="section__images">
+        <div class="section__images-inner">
+          <div class="section__images-wrapper">
+            <slider-gallery data-effect="{{- _section.effect -}}" data-autoplay="{{- _section.autoplay -}}" data-speed="{{- _section.autoplay_speed | times: 1000 -}}">
```

Comment:
Nice! Data attributes for autoplay, speed, and effects is a good detail to pick up on since JS doesn't re-render upon changes in the editor. So doing something like

`let autoplaySpeed = {{ _section.autoplay_speed | times: 1000 }}`

would've caused an issue when adjusting these settings in the editor